megamek/src/megamek/common/options/OptionsConstants.java

```diff
@@ -21,112 +21,111 @@ public class OptionsConstants {
+    public static final String QUIRK_NEG_EXP_ACTUATOR = "exp_actuator"; 
+    public static final String QUIRK_NEG_FLAWED_COOLING = "flawed_cooling"; 
+    public static final String QUIRK_NEG_FRAGILE_FUEL = "fragile_fuel"; 
+    public static final String QUIRK_NEG_GAS_HOG = "gas_hog"; 
+    public static final String QUIRK_NEG_HARD_PILOT = "hard_pilot"; 
+    public static final String QUIRK_NEG_ILLEGAL_DESIGN = "illegal_design"; 
+    public static final String QUIRK_NEG_LARGE_DROPPER = "large_dropper"; 
     public static final String QUIRK_NEG_LOW_ARMS = "low_arms"; 
-    public static final String QUIRK_NEG_NO_EJECT= "no_eject"; 
-    public static final String QUIRK_NEG_NO_TWIST= "no_twist"; 
-    public static final String QUIRK_NEG_NON_STANDARD= "non_standard"; 
-    public static final String QUIRK_NEG_OBSOLETE= "obsolete"; 
+    public static final String QUIRK_NEG_NO_EJECT = "no_eject"; 
+    public static final String QUIRK_NEG_NO_TWIST = "no_twist"; 
+    public static final String QUIRK_NEG_NON_STANDARD = "non_standard"; 
+    public static final String QUIRK_NEG_OBSOLETE = "obsolete"; 
     public static final String QUIRK_NEG_OVERSIZED = "oversized"; 
-    public static final String QUIRK_NEG_POOR_LIFE_SUPPORT= "poor_life_support"; 
-    public static final String QUIRK_NEG_POOR_PERFORMANCE= "poor_performance"; 
-    public static final String QUIRK_NEG_POOR_SEALING= "poor_sealing"; 
+    public static final String QUIRK_NEG_POOR_LIFE_SUPPORT = "poor_life_support"; 
+    public static final String QUIRK_NEG_POOR_PERFORMANCE = "poor_performance"; 
+    public static final String QUIRK_NEG_POOR_SEALING = "poor_sealing"; 
     public static final String QUIRK_NEG_POOR_TARG_L = "poor_target_long"; 
     public static final String QUIRK_NEG_POOR_TARG_M = "poor_target_med"; 
     public static final String QUIRK_NEG_POOR_TARG_S = "poor_target_short"; 
-    public static final String QUIRK_NEG_POOR_WORK= "poor_work"; 
-    public static final String QUIRK_NEG_PROTOTYPE= "prototype"; 
-    public static final String QUIRK_NEG_RAMSHACKLE= "ramshackle"; 
-    public static final String QUIRK_NEG_SENSOR_GHOSTS= "sensor_ghosts"; 
-    public static final String QUIRK_NEG_SUSCEPTIBLE_CWS= "susceptible_cws"; 
-    public static final String QUIRK_NEG_UNBALANCED= "unbalanced"; 
-    public static final String QUIRK_NEG_UNSTREAMLINED= "unstreamlined"; 
+    public static final String QUIRK_NEG_POOR_WORK = "poor_work"; 
+    public static final String QUIRK_NEG_PROTOTYPE = "prototype"; 
+    public static final String QUIRK_NEG_RAMSHACKLE = "ramshackle"; 
+    public static final String QUIRK_NEG_SENSOR_GHOSTS = "sensor_ghosts"; 
+    public static final String QUIRK_NEG_SUSCEPTIBLE_CWS = "susceptible_cws"; 
+    public static final String QUIRK_NEG_UNBALANCED = "unbalanced"; 
+    public static final String QUIRK_NEG_UNSTREAMLINED = "unstreamlined"; 
     public static final String QUIRK_NEG_WEAK_HEAD_1= "weak_head_1"; 
```

Comment:
Forgot some :D

Reply:
I did not expect game options with names ending with numbers, so I missed them. Fixed.